File: server/simpleaide/templates/base-node/files/src/routes/health.ts
import { Router } from "express";

const router = Router();

router.get("/", (req, res) => {
  res.json({
    status: "healthy",
    timestamp: new Date().toISOString(),
    uptime: process.uptime()
  });
});

router.get("/ready", (req, res) => {
  res.json({ ready: true });
});

export default router;

File: server/simpleaide/templates/base-python/files/src/routes/health.py
import time
from fastapi import APIRouter

router = APIRouter()
start_time = time.time()


@router.get("/")
async def health():
    return {
        "status": "healthy",
        "uptime": time.time() - start_time
    }


@router.get("/ready")
async def ready():
    return {"ready": True}
